ports/winit/app.rs

```diff
@@ -29,6 +34,62 @@ pub struct App {
     event_queue: RefCell<Vec<EmbedderEvent>>,
     suspended: Cell<bool>,
     windows: HashMap<WindowId, Rc<dyn WindowPortsMethods>>,
+    minibrowser: Option<RefCell<Minibrowser>>,
+}
+
+struct Minibrowser {
```

Comment:
Perhaps this could move to a separate file just for cleanliness? Maybe `minibrowser.rs`.

Reply:
done in 06e0d2987e42a1fd542a34697be3f7ca37ecdc62